tests/test_cli_cicd_boost.py

```
@@ -0,0 +1,202 @@
+"""Tests for cli/cicd.py – generate_cicd and feedback commands."""
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)